qiita_db/test/test_search.py

```diff
@@ -84,12 +84,14 @@ def test_parse_study_search_string(self):
         st_sql, samp_sql, meta = \
             self.search._parse_study_search_string(
                 'host_subject_id includes "Chicken little"')
-        exp_st_sql = "SELECT study_id FROM qiita.study_sample_columns"
-        exp_samp_sql = ("SELECT r.sample_id,r.host_subject_id FROM "
-                        "qiita.required_sample_info r JOIN qiita.sample_{0} sa"
+        exp_st_sql = ("SELECT study_id FROM qiita.study_sample_columns "
+                      "WHERE lower(column_name) = lower('host_subject_id') "
```

Comment:
That is actually built programmatically, and lowers everything it gets so that search is case-insensitive.